build.gradle.kts

```diff
@@ -9,31 +9,26 @@ description = "The first Static Analysis and REMEDIATION tool"
 
 val rewriteVersion = rewriteRecipe.rewriteVersion.get()
 dependencies {
-    compileOnly("org.projectlombok:lombok:latest.release")
-    annotationProcessor("org.projectlombok:lombok:latest.release")
-    testImplementation("org.projectlombok:lombok:latest.release")
-
-    implementation(platform("org.openrewrite:rewrite-bom:${rewriteVersion}"))
-    implementation("org.openrewrite:rewrite-java")
-    implementation("org.openrewrite:rewrite-groovy:${rewriteVersion}")
-    implementation("org.openrewrite:rewrite-kotlin:${rewriteVersion}")
-    implementation("org.openrewrite:rewrite-csharp:${rewriteVersion}")
-    implementation("org.openrewrite.meta:rewrite-analysis:${rewriteVersion}")
-    implementation("org.apache.commons:commons-text:latest.release")
-
     annotationProcessor("org.openrewrite:rewrite-templating:${rewriteVersion}")
```

Comment:
Apply natural ordering to enable grouping. This was revisited due to the investigation into the Java version.
I can extract it of course.

<img width="1784" alt="image" src="https://github.com/user-attachments/assets/1192d99a-062e-47ea-ba90-cfc4baa6e183">

Reply:
Please try to minimize the changes when you open a PR, as that reduces the burden on the reviewer, and any chance of conflicts with parallel work.

Whenever we _do_ want to change such a grouping we typically do so in an automated fashion at scale with recipes, as opposed to manually. That then ensures we're immediately consistent across projects after such a recipe run.

Reply:
indeed, reverted,

Reply:
Obviously its an old java version now even 17?  `org.openrewrite:rewrite-java-17`

```
> Task :compileJava FAILED
/home/runner/work/rewrite-static-analysis/rewrite-static-analysis/src/main/java/org/openrewrite/staticanalysis/EqualsAvoidsNullVisitor.java:50: error: pattern matching in instanceof is not supported in -source 8
        if (superVisitMethodInvocation instanceof J.MethodInvocation methodInvocation) {
```